source/quick-start.txt

```diff
@@ -37,11 +37,62 @@ to check your version of Node and NPM and installation instructions for your
 system, see `downloading and installing Node.js and npm
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
-Initialize the Project
-~~~~~~~~~~~~~~~~~~~~~~
-
 After you verify that you installed the required versions of Node and NPM,
-create a new project.
+create a new project by :ref:`cloning our js-starter repository
+<qs-download-the-repo>` or by :ref:`creating it manually <qs-manual-setup>`.
+
+.. _qs-download-the-repo:
+
+Clone the Project
+~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   This approach requires you to have `Git
+   <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+   installed.
+
+First, clone our `js-starter repository
+<https://github.com/mongodb-university/js-starter>`__  in your command
+line interface using the following command:
+
+.. tabs::
+
+   .. tab:: SSH
+      :tabid: ssh
+      
+      .. code-block:: none
+
+         git clone git@github.com:mongodb-university/js-starter.git 
+
+   .. tab:: HTTPS
+      :tabid: https
+      
+      .. code-block:: none
+
+         git clone https://github.com/mongodb-university/js-starter.git
+
+Then, navigate into that directory so you can work directly with your
+project's files:
+
+.. code-block:: none
+
+   cd js-starter
+
+At this point, you should have appropriate version of Node.js and NPM
+installed as well as a project directory that contains the dependencies you
+need to use the Node.js MongoDB driver.
```

Comment:
```suggestion
The repository contains the following items: 
- Node.js
- NPM
- The MongoDB Node.js driver
```
Suggestion: I think this can be simplified.

Reply:
There really isn't anything in the repo except for a package.json file with the correct dependencies, a list doesn't make sense to me here. But, I'll be adjusting this sentence to remove the mention of NPM.

Reply:
If the driver source code is not in the repo, there may be minimal advantage to having this option. NPM will still be a dependency, as you will still need to download the driver once you clone the repository (this option adds git as a dependency and does not remove the dependency on npm). What is the advantage of cloning the project over creating the project?